proto/buf/alpha/registry/v1alpha1/resolve.proto

```diff
@@ -120,7 +120,7 @@ message LocalModuleResolveResult {
   ResolvedReferenceType resolved_reference_type = 3;
   // Indicates whether this reference is resolvable through BSR_HEAD.
   // False indicates a non-default branch (formerly draft) commit.
-  bool is_bsr_head = 4;
+  bool is_bsr_head = 4 [deprecated = true];
```

Comment:
Are we still using this field?

Reply:
We have a PR open where we stop using it. The RPC still returns it though.

Reply:
Nope, nothing in the UI is using this field.

Reply:
Can we delete and reserve it, similar to other changes we've made?

Reply:
Done